src/cow-react/modules/swap/state/EthFlow/updaters/EthFlowDeadlineUpdater.tsx

```diff
@@ -1,70 +1,85 @@
-import { useEffect, useState } from 'react'
+import { useEffect, useRef } from 'react'
 import { useUserTransactionTTL } from 'state/user/hooks'
 import { useIsEthFlow } from '@cow/modules/swap/hooks/useIsEthFlow'
 import { loadJsonFromLocalStorage, setJsonToLocalStorage } from '@cow/utils/localStorage'
 import { MINIMUM_ETH_FLOW_DEADLINE_SECONDS } from 'constants/index'
+import { DeadlineSettings } from './types'
 
-const LOCAL_STORAGE_KEY = 'UserPreviousDeadline'
+const LOCAL_STORAGE_KEY = 'UserDeadlineSettings'
```

Comment:
Updated to reflect what's stored now and also to avoid issues with former format